src/server/hotkey_collector.h

```diff
@@ -123,5 +126,22 @@ class hotkey_coarse_data_collector : public internal_collector_base
     std::vector<std::atomic<uint64_t>> _hash_buckets;
 };
 
+class hotkey_fine_data_collector : public internal_collector_base
+{
+public:
+    hotkey_fine_data_collector() = delete;
```

Comment:
```suggestion
    hotkey_fine_data_collector() = delete;
```

move deleted to private